CHANGELOG.md

```diff
@@ -57,7 +57,7 @@ There are no major changes in this release, although JSON & Plist template write
 * Avoid breaking the system swift installation when installing SwiftGen via Homebrew on macOS 10.14.4 or higher.  
   [David Jennes](https://github.com/djbe)
   [#686](https://github.com/SwiftGen/SwiftGen/issue/686)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/96c53d24240cdcf7aa236557505ede9b1a207372/CHANGELOG.md#L59"></span>This looks like a GitHub link URL with a typo. Issue links should use <code>/issues/123</code> (plural) and PRs links should use <code>/pull/123</code> (singular).</td>
    </tr>
  </tbody>
</table>